parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -59,14 +60,17 @@ pub type FinalizedHeaderUpdateOf<T> = FinalizedHeaderUpdate<
 >;
 pub type SyncCommitteeOf<T> = SyncCommittee<<T as Config>::MaxSyncCommitteeSize>;
 
+pub type PreparedSyncCommitteeOf<T> =
+	(BoundedVec<PublicKeyPrepared, <T as Config>::MaxSyncCommitteeSize>, PublicKeyPrepared);
```

Comment:
I prefer a proper struct with named fields, instead of a tuple. Is better for auditors and code reviewers:

```
struct SyncCommitteePrepared {
   pubkeys: ...
   aggregate_pubkey: ...
}
```

That also allows you to implement `TryFrom<SyncCommittee>` which further neatens the code.